WordPressCom-Stats-iOS/UI/NSObject+StatsBundleHelper.m

```diff
@@ -0,0 +1,13 @@
+#import "NSObject+StatsBundleHelper.h"
+
+@implementation NSObject (StatsBundleHelper)
+
+- (NSBundle *)statsBundle
+{
+    NSString *path = [[NSBundle mainBundle] pathForResource:@"WordPressCom-Stats-iOS" ofType:@"bundle"];
```

Comment:
`mainBundle` won't work either :)
It gets confusing for nested bundles, but I'll try to explain.
When this is included as a framework:

```
(lldb) po [NSBundle mainBundle]
NSBundle </Users/koke/Library/Developer/CoreSimulator/Devices/0BFE9C53-C0E9-489B-B427-6F33F141490B/data/Containers/Bundle/Application/F05F0BD7-51F5-4D6E-B7CF-858B27C4960E/WordPress.app> (loaded)
(lldb) expr [NSBundle bundleForClass:[WPStatsViewController class]]
(NSBundle *) $9 = 0x00007f80c2418020 @"/Users/koke/Library/Developer/CoreSimulator/Devices/0BFE9C53-C0E9-489B-B427-6F33F141490B/data/Containers/Bundle/Application/F05F0BD7-51F5-4D6E-B7CF-858B27C4960E/WordPress.app/Frameworks/WordPressCom_Stats_iOS.framework"
(lldb) expr [(NSBundle *)$9 pathForResource:@"WordPressCom-Stats-iOS" ofType:@"bundle"]
(__NSCFString *) $10 = 0x00007f80c2f26ac0 @"/Users/koke/Library/Developer/CoreSimulator/Devices/0BFE9C53-C0E9-489B-B427-6F33F141490B/data/Containers/Bundle/Application/F05F0BD7-51F5-4D6E-B7CF-858B27C4960E/WordPress.app/Frameworks/WordPressCom_Stats_iOS.framework/WordPressCom-Stats-iOS.bundle"
(lldb) expr [NSBundle bundleWithPath:$10]
(NSBundle *) $11 = 0x00007f80c2a1ec10 @"/Users/koke/Library/Developer/CoreSimulator/Devices/0BFE9C53-C0E9-489B-B427-6F33F141490B/data/Containers/Bundle/Application/F05F0BD7-51F5-4D6E-B7CF-858B27C4960E/WordPress.app/Frameworks/WordPressCom_Stats_iOS.framework/WordPressCom-Stats-iOS.bundle"
```

The first `NSBundle` is not your bundle but the bundle where your bundle is. It used to be the main app, because resources were just copied, but now it lives inside a framework, which is another kind of bundle.